_posts/2021-03-03-js-onboarding.md

```diff
@@ -0,0 +1,70 @@
+---
+author: Eder Soares
+layout: article
+title: "Onboarding JS"
+date: 2021-03-03 18:00:00 -0300 
+categories: JS
+thumb: sample.jpg
+language: pt_br
+---
+
+Este tutorial tem como objetivo proporcionar um nivelamento de conhecimento na linguagem javascrit.
+
+## Sumário
+
+1. Requisitos
+2. Links úteis
+3. Passos a serem seguidos
+4. Passos sugeridos
+
+## 1. Requisitos
+
+- Acesso a Udemy
+- Lógica de programação
+
+## 2. Links úteis
+
+No link a seguir você encontrará o curso completo que servirá de base para essa capacitação.
+
+[Udmey - Curso Web Moderno Completo com JavaScript 2021 + Projetos](https://www.udemy.com/course/curso-web){:target="_blank"}
+
+Recomendamos também guardar este link para consultar caso tenha dúvidas nas implementações:
+
+[W3schools - JavaScript Tutorial](https://www.w3schools.com/js/default.asp){:target="_blank"}
+
+## 3. Passos a serem seguidos
+
+Nos links a seguir, estarão listados as aulas ou sessões que abordam conceitos essenciais, sinta-se livre para explorar o restante do curso.
+
+1. Fundamentos
```

Comment:
What do you think about we add the "O Básico de Var, Let e Const" class too? I've seen people with doubts about how scope works

Reply:
As we discuss it is already mentioned in "Função" section